src/diagrams/er/parser/erDiagram.jison

```diff
@@ -28,7 +28,7 @@ accDescr\s*"{"\s*                                { this.begin("acc_descr_multili
 "erDiagram"                     return 'ER_DIAGRAM';
 "{"                             { this.begin("block"); return 'BLOCK_START'; }
 <block>\s+                      /* skip whitespace in block */
-<block>(?:PK)|(?:FK)            return 'ATTRIBUTE_KEY'
+<block>\b((?:PK)|(?:FK))\b      return 'ATTRIBUTE_KEY'
```

Comment:
What does the `\b`means ?

Reply:
`\b` is word boundary. So it ensures `FK` or `PK` is an standalone word.

Reply:
OK